runtime/src/constants.rs

```diff
@@ -35,13 +35,12 @@ pub mod asset {
 /// Money matters.
 pub mod currency {
 	use cennznet_primitives::types::Balance;
-
-	pub const PICOCENTS: Balance = 1;
-	pub const NANOCENTS: Balance = 1_000 * PICOCENTS;
-	pub const MICROCENTS: Balance = 1_000 * NANOCENTS;
-	pub const MILLICENTS: Balance = 1_000 * MICROCENTS;
-	pub const CENTS: Balance = 1_000 * MILLICENTS; // assume this is worth about a cent.
-	pub const DOLLARS: Balance = 100 * CENTS;
+	/// The smallest denomination of any currency
+	pub const WEI: Balance = 1;
+	/// The smallest denomination of a 4dp currency
```

Comment:
Could be better if we hadn't coupled these definitions with 4dp which is configurable for an asset.